cross/libsoxr/Makefile

```diff
@@ -0,0 +1,29 @@
+PKG_NAME = soxr
+PKG_VERS = 0.1.2
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS)-Source.$(PKG_EXT)
+PKG_DIST_SITE = https://sourceforge.net/projects/soxr/files
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)-Source
+DEPENDS =
+
+HOMEPAGE = https://sourceforge.net/projects/soxr/
+COMMENT  = The SoX Resampler library 'libsoxr' performs one-dimensional sample-rate conversion—it may be used, for example, to resample PCM-encoded audio.
+LICENSE  = GPL & LGPL
+
+CONFIGURE_TARGET = myConf
```

Comment:
For consistency's sake, I'd prefer to see `myConfigure` throughout this PR. There's several places in this PR that use `myConf`, but nothing in master.